R/plot.earlyR.R

```diff
@@ -90,18 +94,30 @@ plot.earlyR <- function(x, type = c("R", "lambdas"), scale = "ml", ...) {
     lambdas <- scale * (x$lambdas / sum(x$lambdas, na.rm = TRUE))
     df_plot <- data.frame(date = x$dates, lambda = lambdas)
 
-    out <- ggplot2::ggplot(df_plot,
-                           ggplot2::aes(x = .data$date,
-                                        y = .data$lambda)) +
+    if (colour_spectrum) {
+      new_col <- grDevices::colorRampPalette(c("red", "yellow"))(length(lambdas))
+      out <- ggplot2::ggplot(df_plot,
+                             ggplot2::aes(x = .data$date,
+                                          y = .data$lambda,
+                                          fill = new_col)) +
+        ggplot2::scale_fill_manual(values = new_col) +
+        ggplot2::guides(fill = FALSE)
+    } else {
+      out <- ggplot2::ggplot(df_plot,
+                             ggplot2::aes(x = .data$date,
+                                          y = .data$lambda))
+    }
+    out <- out +
+      ggplot2::theme_bw() +
```

Comment:
This makes sense if we want to make the new graphs look like the old ones, but the color on these bars was a very poor design chocie on my end. Do we need this to pass checks?